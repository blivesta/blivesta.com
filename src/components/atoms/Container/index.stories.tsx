import React from 'react';
// import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import Container from './';

storiesOf(`component/atomos/Container`, module).add(`default`, () => <Container>contents</Container>);
