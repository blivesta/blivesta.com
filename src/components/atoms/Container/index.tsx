import styled from 'styled-components';

const Container = styled.div`
  margin: 0 auto;
  max-width: 984px;
  padding: 0px ${props => props.theme.spaces[5]};
  padding-top: 0;
`;

export default Container;
