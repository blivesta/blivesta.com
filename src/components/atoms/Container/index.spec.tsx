import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Container from '.';

describe('Container', () => {
  it('snapshot', () => {
    const tree = shallow(<Container>...</Container>);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
