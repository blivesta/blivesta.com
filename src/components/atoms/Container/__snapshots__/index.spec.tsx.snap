// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  margin: 0 auto;
  max-width: 984px;
  padding: 0px ",
          [Function],
          ";
  padding-top: 0;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  ...
</StyledComponent>
`;
